src/Storages/System/StorageSystemSettingsProfileElements.cpp

```diff
@@ -29,6 +29,7 @@ NamesAndTypesList StorageSystemSettingsProfileElements::getNamesAndTypes()
         {"min", std::make_shared<DataTypeNullable>(std::make_shared<DataTypeString>())},
         {"max", std::make_shared<DataTypeNullable>(std::make_shared<DataTypeString>())},
         {"readonly", std::make_shared<DataTypeNullable>(std::make_shared<DataTypeUInt8>())},
+        {"changeable_in_readonly", std::make_shared<DataTypeNullable>(std::make_shared<DataTypeUInt8>())},
```

Comment:
We can use a single enum field here instead of those two fields.

Reply:
It's a system table, we change system tables sometimes (because they're ours)

Reply:
Let's leave just one column if this is OK